examples/glyph-lcd-cairo.py

```diff
@@ -0,0 +1,115 @@
+
+    face = Face('./Vera.ttf')
+    face.set_char_size( 48*64 )
+    face.load_char('S', FT_LOAD_RENDER |
+                        FT_LOAD_TARGET_LCD )
+    bitmap = face.glyph.bitmap
+    width  = face.glyph.bitmap.width//3
+    rows   = face.glyph.bitmap.rows
+    pitch  = face.glyph.bitmap.pitch
+    whole_thing = (c_ubyte * (pitch * rows))()
+    memmove(pointer(whole_thing), bitmap._FT_Bitmap.buffer,
+            pitch * rows)
+
+    I = ImageSurface(FORMAT_ARGB32, width, rows)
+    ndR = ndarray(shape=(rows,width), buffer=whole_thing,
+                  dtype=ubyte, order='C',
+                  offset=0,
+                  strides=[pitch, 3])
+    ndG = ndarray(shape=(rows,width), buffer=whole_thing,
+                  dtype=ubyte, order='C',
+                  offset=1,
+                  strides=[pitch, 3])
+    ndB = ndarray(shape=(rows,width), buffer=whole_thing,
+                  dtype=ubyte, order='C',
+                  offset=2,
+                  strides=[pitch, 3])
+    ndI = ndarray(shape=(rows,width), buffer=I.get_data(),
+                  dtype=uint32, order='C',
+                  strides=[I.get_stride(), 4])
+    ndI[:,:] = 4278190080 + ndR[:,:] * 65536 + ndG[:,:] * 256 + ndB[:,:]
+    I.mark_dirty()
+
+    surface = ImageSurface(FORMAT_ARGB32, 800, 600)
+    ctx = Context(surface)
+
+    ctx.set_source_surface(I, 0, 0)
+    pattern = ctx.get_source()
+    SurfacePattern.set_filter(pattern, FILTER_BEST)
+    scale = 480.0 / rows
+    scalematrix = Matrix()
+    scalematrix.scale(1.0/scale,1.0/scale)
+    scalematrix.translate(-(400.0 - width *scale /2.0 )+200, -60)
+    pattern.set_matrix(scalematrix)
+    ctx.paint()
+
+    # we need this later for shifting the taller LCD_V glyph up.
+    rows_old = rows
+
+    # LCD_V
+    face.load_char('S', FT_LOAD_RENDER |
+                        FT_LOAD_TARGET_LCD_V )
+    bitmap = face.glyph.bitmap
+    width  = face.glyph.bitmap.width
+    rows   = face.glyph.bitmap.rows//3
+    pitch  = face.glyph.bitmap.pitch
+    whole_thing = (c_ubyte * (pitch * face.glyph.bitmap.rows))()
+    memmove(pointer(whole_thing), bitmap._FT_Bitmap.buffer,
+            pitch * face.glyph.bitmap.rows)
+
+    I = ImageSurface(FORMAT_ARGB32, width, rows)
+    ndR = ndarray(shape=(rows,width), buffer=whole_thing,
+                  dtype=ubyte, order='C',
+                  offset=0,
+                  strides=[pitch*3, 1])
+    ndG = ndarray(shape=(rows,width), buffer=whole_thing,
+                  dtype=ubyte, order='C',
+                  offset=pitch,
+                  strides=[pitch*3, 1])
+    ndB = ndarray(shape=(rows,width), buffer=whole_thing,
+                  dtype=ubyte, order='C',
+                  offset=2*pitch,
+                  strides=[pitch*3, 1])
+    ndI = ndarray(shape=(rows,width), buffer=I.get_data(),
+                  dtype=uint32, order='C',
+                  strides=[I.get_stride(), 4])
+    ndI[:,:] = 4278190080 + ndR[:,:] * 65536 + ndG[:,:] * 256 + ndB[:,:]
```

Comment:
changed my mind - python should cache constant values, so there should be no difference with `256**3`. I'll time it and see.